### PR TITLE
Fix/unmapped events on face liveness

### PR DESCRIPTION
**Pull request by joaogoncalves-onfido**
# Problem

Fixing unmapped legacy event errors.

# Solution

- Remove unused tracking from component;
- Map active video upload events to `FACE_LIVENESS_UPLOAD_PROGRESS*`

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by warrenseine** [comment]
I think there's an `ANALYTICS_CHANGELOG.md` to update. LGTM otherwise.

**Comment by DannyvanderJagt** [comment]
Sorry for the inconvenience , can you also add them to: https://wiki.onfido.net/display/SI/SDK+Analytics+-+List+of+Events+and+Properties

File: src/components/ActiveVideo/ActiveVideo.tsx [hunk 1]
```diff
@@ -115,11 +115,8 @@ const ActiveVideo: FunctionComponent<Props> = (props) => {
...
 }
 
-export default trackComponent(
-  localised(
-    withCrossDeviceWhenNoCamera(
-      withFailureHandling(withPermissionsFlow(ActiveVideo))
-    )
-  ),
-  'face_liveness'
+export default localised(
```
Comment (warrenseine) [inline src/components/ActiveVideo/ActiveVideo.tsx:118]: For some reason, GitHub prevents me from adding comments on lines that are not part of the diff, but there's a warning line 29 you can address.

Comment (joaogoncalves-onfido) [reply src/components/ActiveVideo/ActiveVideo.tsx:117]: You're absolutely right. Unused import for track_component, my bad. Fixed now!

**Review by DannyvanderJagt — APPROVED** [review]

**Pull request merged**